openvpb.js

```diff
@@ -121,21 +126,13 @@ async function getContactDetails() {
                 sidebarContainer.appendChild(container)
 
                 let { yourName, messageTemplates } = await browser.storage.local.get(['yourName', 'messageTemplates'])
-                if (typeof messageTemplates === 'string') {
-                    messageTemplates = JSON.parse(messageTemplates)
-                }
 
                 if (messageTemplates.length > 0) {
                     console.log('Appending button...')
-                    let { label, message, result } = messageTemplates[0]
-
-                    let messageBody = message
-                        .replace(THEIR_NAME_REGEX, contactName)
-                        .replace(YOUR_NAME_REGEX, yourName)
```

Comment:
moved this to launchMessagingApp method so if the message template has changed between the page load the the click event, the latest one will be used.